### PR TITLE
Document CLI Context loader Console

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Document CLI Context loader Console
| Fixed ticket? | Fixes https://github.com/PrestaShop/docs/issues/997

### CODE REVIEW

**Comment by matks** [comment]
Could you please review my contribution @PrestaShop/prestashop-maintainers 😉 ?

File: src/content/1.7/development/components/console/context-helper.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+title: Context helper
+---
+
+# CLI Context helper
+
+The legacy framework of PrestaShop was not built to be run from a Command Line.
+
+The consequence is that many of its functions require specific data to be properly initialized, and these parts are being initialized when a HTTP request is handled by the framework.
```
Comment (kpodemski) [inline src/content/1.7/development/components/console/context-helper.md:9]:
```suggestion
The consequence is that many of its functions require specific data to be properly initialized, and these parts are being initialized when an HTTP request is handled by the framework.
```

File: src/content/1.7/development/components/console/context-helper.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+In order to use legacy classes and components in Console, you need to initialize these parts. To make it easy, a [LegacyContextLoader helper class](https://github.com/PrestaShop/PrestaShop/pull/21125) was introduced in 1.7.8.0 .
```
Comment (kpodemski) [inline src/content/1.7/development/components/console/context-helper.md:11]:
```suggestion
In order to use legacy classes and components in Console, you need to initialize these parts. To make it easy, a [LegacyContextLoader helper class](https://github.com/PrestaShop/PrestaShop/pull/21125) was introduced in version `1.7.8.0`.
```

File: src/content/1.7/development/components/console/context-helper.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+It loads the needed property in `Context` using, when needed, fake Employee or Controller instance.
```
Comment (kpodemski) [inline src/content/1.7/development/components/console/context-helper.md:13]:
```suggestion
It loads the needed property in `Context` using, when needed, a fake Employee or Controller instance.
```

File: src/content/1.7/development/components/console/context-helper.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+It is available as a Symfony service `prestashop.adapter.legacy_context_loader`.
+
+Example
+```
+MyCustomCommand extends ContainerAwareCommand {
```
Comment (kpodemski) [inline src/content/1.7/development/components/console/context-helper.md:19]:
```suggestion
MyCustomCommand extends ContainerAwareCommand
{
```

**Review by kpodemski — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @matks

**Pull request merged**